cubedash/_pages.py

```diff
@@ -197,6 +197,11 @@ def search_page(
         _model.STORE.index.datasets.search(**query, limit=_HARD_SEARCH_LIMIT + 1),
         key=lambda d: d.center_time,
     )
+
+    for i in range(len(datasets)):
+        if _model.STORE.get_dataset_center_time(datasets[i].id):
+            datasets[i].center_time = _model.STORE.get_dataset_center_time(datasets[i].id)
```

Comment:
I’m also a little iffy about overwriting ODC’s own inner data models here directly. The datasets will in some areas of codebase have an Explorer center-time and others an ODC center-time, which might be surprising and unpredictable to future programmers.